translations/ja.json

```
@@ -597,50 +597,50 @@
-	"The \"{{.driver_name}}\" driver requires root privileges. Please run minikube using 'sudo minikube --vm-driver={{.driver_name}}": "「{{.driver_name}}」ドライバにはルート権限が必要です。「sudo minikube --vm-driver={{.driver_name}}」を使用して minikube を実行してください",
-	"The \"{{.driver_name}}\" driver should not be used with root privileges.": "",
-	"The \"{{.name}}\" cluster has been deleted.": "「{{.name}}」クラスタが削除されました",
-	"The \"{{.name}}\" cluster has been deleted.__1": "「{{.name}}」クラスタが削除されました",
-	"The 'none' driver is designed for experts who need to integrate with an existing VM": "",
-	"The 'none' driver provides limited isolation and may reduce system security and reliability.": "ドライバに「none」を指定すると、分離が制限され、システムのセキュリティと信頼性が低下する可能性があります",
-	"The '{{.addonName}}' addon is enabled": "",
-	"The '{{.driver}}' driver requires elevated permissions. The following commands will be executed:\\n\\n{{ .example }}\\n": "",
-	"The '{{.driver}}' provider was not found: {{.error}}": "",
-	"The '{{.name}} driver does not support multiple profiles: https://minikube.sigs.k8s.io/docs/reference/drivers/none/": "",
-	"The '{{.name}}' driver does not respect the --cpus flag": "",
-	"The '{{.name}}' driver does not respect the --memory flag": "",
-	"The --image-repository flag your provided contains Scheme: {{.scheme}}, which will be removed automatically": "",
-	"The --image-repository flag your provided ended with a trailing / that could cause conflict in kuberentes, removed automatically": "",
-	"The CIDR to be used for service cluster IPs.": "サービス クラスタ IP に使用される CIDR",
-	"The CIDR to be used for the minikube VM (virtualbox driver only)": "minikube VM に使用される CIDR（virtualbox ドライバのみ）",
-	"The KVM QEMU connection URI. (kvm2 driver only)": "KVM QEMU 接続 URI（kvm2 ドライバのみ）",
-	"The KVM default network name. (kvm2 driver only)": "",
-	"The KVM driver is unable to resurrect this old VM. Please run `minikube delete` to delete it and try again.": "",
-	"The KVM network name. (kvm2 driver only)": "KVM ネットワーク名（kvm2 ドライバのみ）",
-	"The VM driver crashed. Run 'minikube start --alsologtostderr -v=8' to see the VM driver error message": "",
-	"The VM driver exited with an error, and may be corrupt. Run 'minikube start' with --alsologtostderr -v=8 to see the error": "",
-	"The VM that minikube is configured for no longer exists. Run 'minikube delete'": "",
-	"The \\\"{{.name}}\\\" container runtime requires CNI": "",
-	"The ambassador addon has stopped working as of v1.23.0, for more details visit: https://github.com/datawire/ambassador-operator/issues/73": "",
-	"The apiserver listening port": "API サーバー リスニング ポート",
+	"Target {{.path}} can not be empty": "ターゲット {{.path}} は空にできません",
+	"Test docs have been saved at - {{.path}}": "テストドキュメントは {{.path}} に保存されました",
+	"The \"{{.driver_name}}\" driver requires root privileges. Please run minikube using 'sudo minikube --vm-driver={{.driver_name}}": "\"{{.driver_name}}\" ドライバーにはルート権限が必要です。'sudo minikube --vm-driver={{.driver_name}}'を使用して minikube を実行してください",
```

```suggestion
	"The \"{{.driver_name}}\" driver requires root privileges. Please run minikube using 'sudo minikube --vm-driver={{.driver_name}}": "\"{{.driver_name}}\" ドライバーにはルート権限が必要です。'sudo minikube --vm-driver={{.driver_name}}' を使用して minikube を実行してください",
```
nit: It seems this is not fixed.